cfme/tests/v2v/test_csv_import.py

```diff
@@ -296,12 +295,15 @@ def test_csv_security_group_flavor(appliance, infra_map, valid_vm, provider):
         flavor = provider.data.flavors[1]
     except (AttributeError, KeyError):
         pytest.skip("No provider data found.")
-    content = "Name,Security Group,Flavor\n{valid_vm},{security_group},{flavor}".format(
+    content = "Name,Security Group,Flavor\n{valid_vm},{security_group},{flavor}\n".format(
         valid_vm=valid_vm,
         security_group=security_group,
         flavor=flavor,
     )
-    actual_attributes = {"security_group": security_group, "flavor": flavor}
-    expected_attributes = check_vm_status(appliance, infra_map, actual_attributes, content=content,
+
+    expected_attributes = check_vm_status(appliance, infra_map, content=content,
                                           table_hover=True, security_group=True)
-    assert not DeepDiff(actual_attributes, expected_attributes)
+
+    soft_assert(expected_attributes["security_group"] == security_group)
+    # In some case * appended in flavor name in GUI as a warning which can be safely ignore.
+    assert expected_attributes["flavor"] in (flavor, flavor + " *")
```

Comment:
How about using f-string here instead of string concatenation?
```suggestion
assert expected_attributes["flavor"] in (flavor, f"{flavor} *")
```